app/store/modules/cart/index.js

```diff
@@ -1,10 +1,110 @@
-// store/modules/cart/index.js
+import cartData from './mock/cart';
+import api from './../../utils/api';
+
+/**
+ * Finds the first occurence of item in given cart object
+ *
+ * @param {Object} cart - Cart object
+ * @param {String} sku - Sku of the cart item
+ * @param {String} groupKey - groupKey of the cart item
+ * @returns {Object} - Cart item, if found
+ */
+function findInCart(cart, sku, groupKey) {
+  return cart.items.find(i => i.sku === sku && i.groupKey === groupKey);
+}
 
 export default {
   namespaced: true,
-  state: {},
-  getters: {},
-  mutations: {},
-  actions: {},
-};
+  state: {
+    /**
+     * @type {Object}   Stores cart items
+     */
+    cart: cartData, // TODO - remove mock data once we have the api
+
+    /**
+     * @type {Object} API error
+     */
+    apiError: null,
+  },
+  getters: { },
+  mutations: {
+    /**
+     * Initial cart data provided by spryker
+     *
+     * @param {Object} state - State
+     * @param {Object} data - Cart data
+     */
+    data(state, data) {
+      state.cart = data;
+    },
+
+    /**
+     * Update product item in cart
+     *
+     * @param {Object} state - State
+     * @param {Object} item - Cart item object
+     * @param {string} item.sku - Sku of the product to be added
+     * @param {string} item.groupKey - GroupKey of the product to be added
+     * @param {string} item.quantity - Quantity of the product to be added
+     */
+    updateCartItem(state, item) {
+      const existingItem = findInCart(state.cart, item.sku, item.groupKey);
+
+      console.dir(state.cart.items[0].quantity);
+
+      if (existingItem) {
+        existingItem.quantity = item.quantity;
+      } else {
+        state.cart.items.push(item);
+      }
 
+      console.dir(state.cart);
+    },
+
+    /**
+     * Handles an API failure
+     * @param {Object} state Current state
+     * @param {Object} error API Error that ocured
+     */
+    apiFailure(state, error) {
+      state.apiError = error;
+    }
+  },
+  actions: {
+    /**
+     * Asynchronously add product to cart
+     *
+     * @param {Object} item - Cart item object
+     * @param {string} item.sku - Sku of the product to be added
+     * @param {string} item.groupKey - GroupKey of the product to be added
+     * @param {string} item.quantity - Quantity of the product to be added
+     * @returns {Promise} Promise object
+     */
+    addToCart({ state, commit }, item) {
+      /**
+       * Triggers commit
+       *
+       * @param {Object} response - The response object from backend
+       */
+      const success = function(response) {
+        const responseItem = findInCart(response, item.sku, item.groupKey);
```

Comment:
- [ ] Please note that the response data is always inside a `data` attribute. So it's `response.data`. See https://winterhalter-fenner.atlassian.net/wiki/spaces/WF/pages/241664001/General+structure